zlp-scheduler/app/views/admin/view_term_admin.html.haml

```diff
@@ -18,7 +18,7 @@
                 %tr
                     %td#specialTableData 
                         %form
-                            = link_to c.name, view_cohort_semester_path(c), method: :get, :class => 'button_link'
+                            = link_to c.name, view_cohort_semester_path(c), :class => 'button_link'
```

Comment:
Is there a reason the `method: :get` was removed? Since this was preexisting code, it might be a good idea to leave that there.